### DISCUSSION TITLE
REQUEST: Custom wallpaper per Tag

### DISCUSSION THREAD

**Discussion by JesseWebDotCom** [linuxserver/Heimdall#669] [Ideas]
Ideally we can not only set a wallpaper for the homepage but also wallpaper per tag.   Ex.  I can have a gaming tag with a Mario background and have a movies tag with a Star Wars background.

**Comment by harkers** [comment]
I was thinking, that if its for a Proxmox group, we could have a screen saver with the common commands to administer the server as background - etc
